tensorboard/webapp/feature_flag/store/feature_flag_store_config_provider.ts

```diff
@@ -14,26 +14,21 @@ limitations under the License.
 ==============================================================================*/
 import {InjectionToken} from '@angular/core';
 import {StoreConfig} from '@ngrx/store';
+import {FeatureFlags} from '../types';
+import {FeatureFlagMetadataMap} from './feature_flag_metadata';
 import {FeatureFlagState} from './feature_flag_types';
 
+const defaultFlags = Object.entries(FeatureFlagMetadataMap).reduce(
+  (map, [key, {defaultValue}]) => {
+    map[key] = defaultValue;
+    return map;
+  },
+  {} as Record<string, any>
+) as FeatureFlags;
```

Comment:
This is a pretty gross cast... The issue is that `Object.entries` (and all the other `Object` methods) drops the typing. This cast seems preferable to casting the values but I am open to alternatives.

I COULD just enumerate the feature flags below but my current approach seems to scale better as more feature flags  are added.